_posts/2022-11-18-first-setup.md

```diff
@@ -5,43 +5,58 @@ date: 2022-11-18
 layout: article	
 authors: 
     - kra-mo
+    - kbdharun
 ---
 
 ## Introduction
 
-Congratulations, you now have Vanilla OS installed.
+Congratulations, you have successfully installed Vanilla OS.
 This guide will help you set your system up for the first time.
 
 ## First Setup app
 
-After booting into Vanilla OS for the first time, you will be presented with a First Setup app. This section will guide you through each page in the app.
+After booting into Vanilla OS for the first time, you will be presented with a First Setup app. This section will guide you through each step in detail.
+
+![Welcome](/assets/uploads/first-setup-welcome.webp)
 
 ### Color Scheme
 
 Your preferred color scheme. Default \(meaning that apps decide for themselves\) or Dark. This can be changed later in the Settings app.
 
+![Color Scheme](/assets/uploads/first-setup-color-scheme.webp)
+
 ### Package Manager
 
-Choose your preferred package managers. These are the ways you will be able to install applications. If unsure, leave it on the default (Flatpak). You can install the required dependencies for each one later if you change your mind.
+Choose your preferred package managers. These are the ways you will be able to install applications. If unsure, leave it on the default \(Flatpak\). You can install the required dependencies for each one later if you change your mind.
+
+![Package Manager](/assets/uploads/first-setup-package-manager.webp)
 
 ### Restricted Codecs
 
 This will install popular patented codecs and fonts. Recommended for most users.
```

Comment:
```suggestion
This will install popular patented codecs, fonts and other tools which are restricted due to legal or copyright reasons and aren't present by default in Vanilla OS. (**Recommended for most users.**)
```
Adding this as `unrar` and some other tools are present in the package too. [Ref](https://en.wikipedia.org/wiki/Ubuntu-restricted-extras#Contents)

Reply:
> I will update the wording a little, but I don't want to scare the user with the word "legal"

Sure, I too think we can update it a bit.